low-level/json_msgs/messages/sensors/cpu_data.py

```diff
@@ -38,7 +38,6 @@ class CPUdataMsg(BaseSensorMsg):
     ACTUATOR_MSG_TYPE = "cpu_data"
     MESSAGE_VERSION  = "1.0.0"
 
-    SEVERITY = "warning"

```

Comment:
just retain default severity as 'warning' , and then
>        if self.alert_type == "fault_resolved":
              self.severity = "informational"

Same comment for other resources

Reply:
Done.